### PR TITLE
Add loading animation

### PR DESCRIPTION
**Pull request by HesyRa**
https://github.com/jsdelivr/www.jsdelivr.com/issues/156

### CODE REVIEW

**Comment by jimaek** [comment]
Very broken https://dl.dropboxusercontent.com/s/sxant2nlzmmkivw/chrome_2018-02-07_15-45-30.png

**Comment by jimaek** [comment]
It should also continue spinning until all elements load. Right now the main spinner disappeared once the title loaded but it had to continue until we had a file list.

**Comment by MartinKolarik** [comment]
@HesyRa the animation will probably need to be controlled by JS (adding and removing a CSS class) and should be visible until we get [here](https://github.com/jsdelivr/www.jsdelivr.com/blob/master/src/views/components/package.html#L211).

File: src/public/less/pages/package.less [hunk 1]
```diff
@@ -7,5 +7,18 @@
 
 	.c-slider {
 		overflow: hidden;
+		.c-slide {
```
Comment (MartinKolarik) [inline src/public/less/pages/package.less]: Change this to `.c-package` (and move the code to `less/components/c-package` to fix the issue @jimaek mentioned.

Comment (HesyRa) [reply src/public/less/pages/package.less]: we are thinking of trying other method. thnx

**Comment by jimaek** [comment]
@MartinKolarik Please check

**Pull request merged**